vaccine_feed_ingest/runners/ky/govstatus/normalize.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+_source_name = "ky:govstatus"
+
+
+def _get_id(site: dict) -> str:
+    return site["id"]
+
+
+def _get_name(site: dict) -> str:
+    return site["name"]
+
+
+def _get_city(site: dict) -> str:
+    return site["address"]["city"]
+
+
+def _get_address(site: dict):
+    return schema.Address(
+        street1=site["address"]["street1"],
+        city=_get_city(site),
+        zip=site["address"]["zip"],
+        state=site["address"]["state"],
+    )
+
+
+def _get_location(site: dict):
```

Comment:
return type is `Optional[schema.LatLng]`

Reply:
Added.